File: languages/python/bloock/entity/availability/loader_args.py
import bloock._bridge.proto.availability_entities_pb2 as proto


class LoaderArgs:
    """
    Represents the arguments for a data loader.
    """
    def __init__(self, id: str) -> None:
        """
        Constructs a LoaderArgs object with the specified parameters.
        :type id: object
        :rtype: object
        """
        self.id = id
        """
        Is a unique identifier associated with the loader.
        """

    def to_proto(self) -> proto.LoaderArgs:
        return proto.LoaderArgs(id=self.id)

File: languages/php/Bloock/Entity/Availability/LoaderArgs.php
<?php

namespace Bloock\Entity\Availability;

/**
 * Represents the arguments for a data loader.
 */
class LoaderArgs
{
    private string $id;

    /**
     * Constructs a LoaderArgs object with the specified parameters.
     * @param string $id
     */
    public function __construct(string $id)
    {
        $this->id = $id;
    }

    public function toProto(): \Bloock\LoaderArgs
    {
        $p = new \Bloock\LoaderArgs();
        $p->setId($this->id);
        return $p;
    }
}